lib/dal/src/attribute/value.rs

```diff
@@ -687,6 +689,11 @@ impl AttributeValue {
                                         .value,
                                 ]
                             }
+                            ValueSource::Secret(secret_id) => {
+                                vec![Secret::payload_for_prototype_execution(ctx, secret_id)
+                                    .await
+                                    .map_err(Box::new)?]
+                            }
```

Comment:
This is where the payload is used for the sole argument to the identity function. This is only used for _direct_ dependent `AttributeValues`.